linux_pipeline/Jenkinsfile_developer_comparison_validation

```diff
@@ -14,10 +14,10 @@ properties ([
             separatorStyle: "",
             sectionHeader: "KERNEL BUILD CONFIGURATION",
             sectionHeaderStyle: env.PARAM_SEPARATOR_STYLE],
-        [$class: 'ChoiceParameterDefinition',
+        [$class: 'StringParameterDefinition',
             name: 'KERNEL_GIT_URL',
-            choices: """git://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git\nhttps://git.kernel.org/pub/scm/linux/kernel/git/davem/net-next.git\nazure_kernel""",
-            description: 'Kernel tree repo'],
+            defaultValue: 'git://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git',
+            description: 'git url or azure_kernel'],
```

Comment:
can you add the old choice values in the description, so that the pipeline user can get these options quickly